app/messages.py

```diff
@@ -218,7 +218,7 @@
     "message": "Mentorship" " relation was" " rejected successfully."
 }
 MENTORSHIP_RELATION_WAS_CANCELLED_SUCCESSFULLY = {
-    "message": "Mentorship" " relation was " "cancelled successfully"
+    "message": "Mentorship" " relation was " "cancelled successfully."
```

Comment:
```suggestion
    "message": "Mentorship relation was cancelled successfully."
```

the double quotes between the whole message seem unnecessary 🤔 
I know that did not came from you, but it can easily be fixed here ;)

Reply:
For sure, happy to do it!